### ISSUE TITLE
關於ISSBA訓練會突然終止

### ISSUE THREAD

**Issue by BaCHiiii** [THUYimingLi/BackdoorBox#91]
![Image](https://github.com/user-attachments/assets/0f0d194e-dd62-4339-85e3-b327e40e83bc)
想請問我再跑test_ISSBA.py時，雖然前面會正常跑，但跑到一半就會停下來不跑了(直接卡住)，請問你們有遇過嗎?是因為batch size太大嗎?

**Comment by BaCHiiii** [comment]
另外，我想請問你們能否提供Blind的訓練參數，因為我目前測試了許多都train不太起來

**Comment by LandAndLand** [comment]
> Blind
你好，我们使用的是原论文中的训练参数：
Training details. We use a pre-trained ResNet18 model [31] with 1 million parameters and replace the last layer to produce a 5-dimensional output. We train for 10 epochs with the Adam optimizer, batch size 64, and learning rate 10−5
.

**Comment by LandAndLand** [comment]
> ![Image](https://github.com/user-attachments/assets/0f0d194e-dd62-4339-85e3-b327e40e83bc) 想請問我再跑test_ISSBA.py時，雖然前面會正常跑，但跑到一半就會停下來不跑了(直接卡住)，請問你們有遇過嗎?是因為batch size太大嗎?

你好，我们暂时没有遇到这个问题。或许你可以尝试调试一下batch size看看是不是程序被kill掉了

**Comment by BaCHiiii** [comment]
> > Blind
> > 你好，我们使用的是原论文中的训练参数：
> > Training details. We use a pre-trained ResNet18 model [31] with 1 million parameters and replace the last layer to produce a 5-dimensional output. We train for 10 epochs with the Adam optimizer, batch size 64, and learning rate 10−5
> > .

好的謝謝。

**Comment by BaCHiiii** [comment]
> > ![Image](https://github.com/user-attachments/assets/0f0d194e-dd62-4339-85e3-b327e40e83bc) 想請問我再跑test_ISSBA.py時，雖然前面會正常跑，但跑到一半就會停下來不跑了(直接卡住)，請問你們有遇過嗎?是因為batch size太大嗎?
> 
> 你好，我们暂时没有遇到这个问题。或许你可以尝试调试一下batch size看看是不是程序被kill掉了

這部分我有跑成功了，謝謝!
